src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -33,8 +34,9 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
```

Comment:
You have much more request parameters in your application than one. Fix it for all other endpoints 
![image](https://user-images.githubusercontent.com/32339836/236231535-db3150b5-3ed6-412b-8215-f070f1732e8c.png)